department-of-veterans-affairs/vets-website
# Add component for post form field content in VAOS

## Description
This PR
- Adds a new component that is just a div with a tabindex, so that we can wrap content after form fields with something in the tab order
- Updates pages in our form flows that need a tab stop in order for screen reader users to be notified of content

How to test:
- Go to http://localhost:3001/health-care/schedule-view-va-appointments/appointments/
- Go through form flows for new appointment and express care and verify that alerts after form fields are in the tab order

## Testing done
Local testing

## Acceptance criteria
- [ ] Content after form fields is not skipped by screen reader users using tab

## Definition of done
